handlers/api.go

```diff
@@ -4095,12 +4095,12 @@ func adjustQueryResults(data []interface{}, adjustQueryEntriesFuncs ...func(map[
 	return nil
 }
 
-// SendErrorResponse exposes sendErrorResponse
-func SendErrorResponse(w http.ResponseWriter, route, message string) {
-	sendErrorResponse(w, route, message)
+// SendBadRequestResponse exposes sendErrorResponse
+func SendBadRequestResponse(w http.ResponseWriter, route, message string) {
+	sendBadRequestResponse(w, route, message)
 }
 
-func sendErrorResponse(w http.ResponseWriter, route, message string) {
+func sendBadRequestResponse(w http.ResponseWriter, route, message string) {
 	sendErrorWithCodeResponse(w, route, message, 400)
```

Comment:
**Nitpick:** I know this does not come from your PR but as you have touched this function I suggest to replace the hardcoded 400 with `http.StatusBadRequest`.

Related: https://github.com/gobitfly/eth2-beaconchain-explorer/blob/77cf0bef9cdc9d872321f989479eca66e4ce334c/handlers/api.go#L4107-L4109
and `http.StatusInternalServerError`.

Reply:
fix